pkg/kubelet/server/server.go

```diff
@@ -225,7 +225,8 @@ func (s *Server) InstallAuthFilter() {
 		// Authenticate
 		u, ok, err := s.auth.AuthenticateRequest(req.Request)
 		if err != nil {
-			glog.Errorf("Unable to authenticate the request due to an error: %v", err)
+			glog.Errorf("Unable to authenticate the request, from %v for %v, due to an error: %v",
```

Comment:
What scenario causes us to get this in a place where the client doesn't know who he is and what he's requesting?  We have this information in RBAC logs because a cluster-admin or developer has to take action (modify a role) for controllers. I don't see the congruent action for our programmatic accessors today.